tests/NationalInstruments.Analyzers.UnitTests/LongChainDottedInvocationAnalyzerTests.cs

```diff
@@ -0,0 +1,331 @@
+using System.Collections.Generic;
+using NationalInstruments.Analyzers.Style;
+using NationalInstruments.Analyzers.TestUtilities;
+using NationalInstruments.Analyzers.TestUtilities.TestFiles;
+using NationalInstruments.Analyzers.TestUtilities.Verifiers;
+using Xunit;
+
+namespace NationalInstruments.Analyzers.UnitTests
+{
+    public class LongChainDottedInvocationAnalyzerTests : NIDiagnosticAnalyzerTests<LongChainDottedInvocationAnalyzer>
```

Comment:
Do we have a test case that is using dotted properties along with dotted methods? We only want to create a diagnostic if you have multiple dotted methods with lamdbas as opposed to properties or methods without parameters, since it's the lamdbas that are the huge readability issue.

Reply:
Yes. [Here](https://github.com/ni/csharp/blob/users/tsharma/add-long-dotted-chain-analyzer/tests/NationalInstruments.Analyzers.UnitTests/ChainOfMethodsWithLambdasAnalyzerTests.cs#L235) and [here](https://github.com/ni/csharp/blob/users/tsharma/add-long-dotted-chain-analyzer/tests/NationalInstruments.Analyzers.UnitTests/ChainOfMethodsWithLambdasAnalyzerTests.cs#L331). One emits a diagnostic, other does not.